test/numerical/TestLSTM.cpp

```diff
@@ -70,14 +80,23 @@ int main(int argc, char *argv[]) {
     // Whether initial value of the cell(initial_c) is specified.
     const auto isNoneC = *rc::gen::element(0, 1);
     // Whether the weight tensor for peepholes(P) is specified.
+#ifdef TEST_NONEP_ONLY
```

Comment:
Suggest to use `TEST_LSTM_NO_P` or `TEST_LSTM_NO_PEEPHOLE` to make it consistent with the other environment variables.

Reply:
Done. Thanks!